tests/observability/scripts/perform_kiali_test.txt

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Setup port-forwarding to Kiali pod as it is not accessible from outside
+# the cluster, but commands are run from outside the cluster.
+function setup_kiali_port_forwarding() {
+  local local_port=$1
+  local kiali_pod=$(kubectl get pod -l app=kiali -n istio-system | grep 'kiali' | head -1 | awk '{ print $1 }')
+  local kiali_port=$(kubectl get service kiali -n istio-system -o jsonpath='{.spec.ports[?(@.name=="http-kiali")].targetPort}')
+
+  echo "Setting up port-forwarding for Kiali at local port $local_port"
+  kubectl port-forward $kiali_pod $local_port:$kiali_port -n istio-system &
+  KIALI_PORT_FORWARD_PID=$!
+  KIALI_LOCAL_URL="http://localhost:$local_port/kiali/api"
+}
+
+function cleanup_kiali_port_forwarding() {
+  local exit_code=$?
+  echo "Exited with $exit_code"
+  echo "Stopping port-forward for Kiali"
+  if [ -z $KIALI_PORT_FORWARD_PID ]; then
+    echo 'It seems that Kiali port-forward PID is not set up properly'
+    kill $(netstat -ntpl | grep '127.0.0.1:8888' | awk '{ print $7 }' | cut -d'/' -f1)
+  else
+    kill $KIALI_PORT_FORWARD_PID
+  fi
+  exit $exit_code
+}
+
+## Setup port-forwarding for Kiali and register trap
```

Comment:
Change "Setup" to "Set up".  (https://developers.google.com/style/word-list#setup)